### DISCUSSION TITLE
Fehler beim Hochladen der Firmware Image in den RAM

### DISCUSSION THREAD

**Discussion by fskander** [Freetz-NG/freetz-ng#343] [Allgemein] [answered]
Hallo,

Erstmal Entschuldigung im Voraus wenn ich Fehler mache, Deutsch ist für mich eine Fremdsprache ;)

Ich möchte Freetz auf meinem Fritzbox 7530 installieren, u.a um OpenVPN und/oder WireGuard nutzen zu können.
Ich könnte die Freetz Image [mithilfe dieser Anleitung](https://freetz-ng.github.io/freetz-ng/wiki/10_Beginner/newbie.de.html ) erstellen, ich habe aber Probleme wenn es kommt zu flashen.
Die angegebene Methode (Upload uber die AVM Interface) funktioniert nicht mehr, ich habe ein paar Recherche gemacht und ich bin auf [diese Anleitung](https://web.archive.org/web/20190529205946/https://www.ip-forum.eu/howto-fritz-box-firmware-images-auch-unsignierte-ueber-den-bootloader-installieren-577) aufmerksam geworden.

- Ich bin auf Windows10 und ich habe PowerShell v7.1 heruntergeladen
- Der Bootloader könnte ich mit der 1. Methode (Ethernet) erreichen und könnte mich auch einloggen
- Firmware Image erstellen/vorbereiten : OK
- Der Fehler kommt wenn ich versuche die Firmware Image in den RAM hochzuladen. Wenn ich die folgende Kommandozeile in PowerShell eingebe :
```
c:\YourFritz\master\eva_tools\EVA-FTP-Client.ps1 -Verbose -Debug -ScriptBlock { BootDeviceFromImage c:\YourFritz\Images\firmware.image.in-memory }
0
```
Bekomme ich dann die folgende Fehlermeldung :
```
"Exception calling ".ctor" with "2 argument(s): "No connection could be made because the target machine actively refused it. 192.168.178.1:21".
Error connecting to remote host
```
<a href="https://ibb.co/wBFzQwN"><img src="https://i.ibb.co/YPFLdy7/2021-07-04-1.png" alt="2021-07-04-1" border="0"></a>

Ich habe mein Firewall (Windows Defender) deaktiviert, es hat sich nichts geändert. Was mache ich falsch ?

**Comment by fda77** [comment]
Du kannst auch in englisch schreiben.
Falls @PeterPawn hier nicht antwortet, hier nachfragen: https://github.com/PeterPawn/YourFritz Diese Windowsscripte hab ich nie benutzt

**Reply by fskander** [reply]
Danke für die Antwort - ich habe dort auch nachgefragt.
Wie hättest du das sonst gemacht ? Ich bin offen für Alternative - solange es funktioniert ;)

**Reply by fda77** [reply]
See readme

**Comment by fda77** [comment, marked as answer]
https://github.com/PeterPawn/YourFritz/discussions/40
